src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,74 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import core.basesyntax.exception.CustomException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService service;
+
+    @BeforeAll
+    static void beforeAll() {
+        service = new RegistrationServiceImpl();
```

Comment:
Consider whether `@BeforeAll` is the appropriate choice for initializing the `RegistrationServiceImpl` instance. If the `RegistrationService` is stateless, it's fine. However, if any state could be shared between tests, `@BeforeEach` should be used to ensure test isolation.